### PR TITLE
Fixes Android Lint Errors

### PR DESCRIPTION
**Pull request by nesalang**
fixes lint errors that cause build failures

### CODE REVIEW

File: source/android/mobile/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
             android:configChanges="orientation|screenSize|keyboardHidden">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
-
+                <action android:name="android.media.action.MEDIA_PLAY_FROM_SEARCH" />
```
Comment (riarenas) [inline source/android/mobile/src/main/AndroidManifest.xml]: What is this for? I thought this was for apps that want to play music.

Comment (nesalang) [reply source/android/mobile/src/main/AndroidManifest.xml]: "To support voice searches on Android Auto, you should also register an intent-filter for the action android.media.action.MEDIA_PLAY_FROM_SEARCH."

---
In reply to: [135900329](https://github.com/Microsoft/AdaptiveCards/pull/588#discussion_r135900329) [](ancestors = 135900329)

File: source/android/adaptivecards/build.gradle [hunk 2]
```diff
@@ -29,6 +30,19 @@ android {
     }
 }
 
+uploadArchives {
+        repositories.mavenDeployer {
+            repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2") {
+                authentication(userName: "adaptivecards", password: "Sird[oshkIt5")
```
Comment (riarenas) [inline source/android/adaptivecards/build.gradle]: having the password there seems like a very bad idea.

Comment (riarenas) [reply source/android/adaptivecards/build.gradle]: And please change it ASAP before adding the new commits.

**Pull request closed without merging**